Reference/google/protobuf/any_test.pb.swift

```diff
@@ -107,6 +107,7 @@ struct ProtobufUnittest_TestAny: SwiftProtobuf.Proto3Message, SwiftProtobuf._Mes
   init() {}
 
   mutating func _protoc_generated_decodeMessage<D: SwiftProtobuf.Decoder>(decoder: inout D) throws {
+    _ = _uniqueStorage()
     try withExtendedLifetime(_storage) { (_storage: _StorageClass) in
```

Comment:
That's exactly what I've been thinking, which is what makes it so puzzling that the compiler would generate a retain/release for every access instead of using the ownership information that it should already have.